quesma/queryparser/pancake_json_rendering.go

```diff
@@ -173,18 +190,25 @@ func (p *pancakeJSONRenderer) combinatorBucketToJSON(remainingLayers []*pancakeM
 }
 
 func (p *pancakeJSONRenderer) layerToJSON(remainingLayers []*pancakeModelLayer, rows []model.QueryResultRow) (model.JsonMap, error) {
-
 	result := model.JsonMap{}
 	if len(remainingLayers) == 0 {
 		return result, nil
 	}
+
 	layer := remainingLayers[0]
+
 	for _, metric := range layer.currentMetricAggregations {
 		metricRows := p.selectMetricRows(metric.internalName+"_col_", rows)
 		result[metric.name] = metric.queryType.TranslateSqlResponseToJson(metricRows, 0) // TODO: fill level?
 		// TODO: maybe add metadata also here? probably not needed
 	}
 
+	// pipeline aggregations of metric type behave just like metric
+	for metricPipelineAggrName, metricPipelineAggrResult := range p.pipelinesProcessor.calculateThisLayerMetricPipelines(layer, rows) {
```

Comment:
We may consider simpler names. Such as p.pipeline.currentPipeloneMetricAgggregation